launcher/sdw_notify/Notify.py

```diff
@@ -16,19 +16,6 @@
 LAST_UPDATED_FILE = os.path.join(BASE_DIRECTORY, "sdw-last-updated")
 LAST_UPDATED_FORMAT = "%Y-%m-%d %H:%M:%S"
 
-# The lockfile basename used to ensure this script can only be executed once.
-# Default path for lockfiles is specified in sdw_util
-LOCK_FILE = "sdw-notify.lock"
```

Comment:
In truth, package users should disregard these variables - they're intended for developer use, defined as shared pseudo-constants to make them easier to modify in one place. If we ever expose any aspect of the updater to user configuration, it would likely be via the configuration file `config.json` in `/usr/share/securedrop-workstation-dom0-config` in `dom0`.

Reply:
By "package users" I meant devs, not end-users! Sorry my wording was unclear. In my mind the "package" being a Python concept is used by devs. Anyhow, I agree in not thinking there is much/anything we'd want end-users (SecureDrop admins?) to configure in the foreseeable future. (The workstation is probably complex enough with reasonably set defaults!)

Reply:
This is not an essential part of the changes, and is definitely the part I thought might be contentious ;) I'm open to reverting it, and/or changing the name of the file if that's the sticking point. (Condensed rationale for splitting out the variables into a separate file is in the PR description)

Reply:
Certainly **not** a sticking point!

Clarification: I was trying to find out what package name would best describe what constants are expected in it, because I think the rationale for splitting them apart makes sense and I anticipate we're likely to want to repeat that in the future. At that point, the question will likely be: do this constant and that one belong together or should they be in distinct files? At that point I feel that `Const` as a name doesn't help much making distinctions.

Let's cross that naming bridge when we get there!